src/main/java/uk/gov/digital/ho/hocs/queue/migration/MigrationData.java

```diff
@@ -1,18 +1,21 @@
 package uk.gov.digital.ho.hocs.queue.migration;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jayway.jsonpath.*;
```

Comment:
We should avoid global imports as it prevents unintended behaviour and bringing in classes we won't use.

Reply:
I've removed the global import and made it specific.